py/picca/delta_extraction/survey.py

```diff
@@ -139,6 +197,7 @@ def load_config(self, config_file):
         """
         # load configuration
         self.config = Config(config_file)
+        self.num_processors = int(self.config.num_processors)
```

Comment:
can we keep this as an `int` directly in the configuration? In the documentation it says it is so, but line 476 in config.py needs to be modified to `self.num_processors = section.getint("num processors")` for things to be consistent